_includes/releases/v22.1/v22.1.12.md

```diff
@@ -0,0 +1,104 @@
+- Fixed a bug that resulted in some retriable errors not being retried during IMPORT. [#90432][#90432]
+- Fixed a bug in `Concat` projection operators for arrays that could cause non-null values to be added to the array when one of the arguments was null. [#91653][#91653]
+- Fixed a bug described in #91069. [#91089][#91089]
+- Fixed a bug introduced in 21.2 that could cause an internal error in rare cases when a query required a constrained index scan to return results in order. [#91692][#91692]
+- Fixed a bug which, in rare cases, could result in a changefeed missing rows which occur around the time of a split in writing transactions which take longer than the closed timestamp target duration (defaults to 3s). [#91749][#91749]
+- Add leading zeros to fingerprint IDs with less than 16 characters. [#91959][#91959]
+- Fixed a bug introduced in 20.2 that could cause filters to be dropped from a query plan with many joins in rare cases. [#91654][#91654]
+- Fixed an unhandled error that could happen if `ALTER DEFAULT PRIVILEGES` was run on the system database. [#92083][#92083]
+- Reduced the amount that RESTORE over-splits. [#91141][#91141]
+- Fixed a bug causing changefeeds to fail when a value is deleted while running on a non-primary column family with multiple columns. [#91953][#91953]
+- Strip quotation marks from database and table names to correctly query for index usage statistics. [#92282][#92282]
+- Fixed the statement activity page so that it no longer shows multi-statement implicit transactions as "explicit." [#92430][#92430]
+- Fixed a bug existing since 20.2 that could cause incorrect results in rare cases for queries with inner joins and left joins. For the bug to occur, the left join had to be in the input of the inner join and the inner join filters had to reference both inputs of the left join, and not filter NULL values from the right input of the left join. Additionally, the right input of the left join had to contain at least one join, with one input not referenced by the left join's ON condition. [#92103][#92103]
+- When configured to true, the `sql.metrics.statement_details.dump_to_logs` cluster setting no longer causes a mutex deadlock. [#92278][#92278]
+- Fixdc incorrect cancellation logic when attempting to detect stuck range feeds. [#92702][#92702]
+- Fixed an internal error when comparing a tuple type with a non-tuple type. [#92714][#92714]
+- `attidentity` for `GENERATED BY DEFAULT AS IDENTITY` column should be `d`.  <what was there before: Previously, ...> <why it needed to change: This was inadequate because ...> <what you did about it: To address this, this patch ...> [#92835][#92835]
+- CockroachDB previously could incorrectly evaluate queries that performed left semi and left anti "virtual lookup" joins on tables in `pg_catalog` or `information_schema`. These join types can be planned when a subquery is used inside of a filter condition. The bug was introduced in 22.1.0 and is now fixed. [#92881][#92881]
```

Comment:
Nit: maybe move "previously" here to the start of the sentence? 

And add, "v22.1.0"